content/rs/installing-upgrading/configuring/change-location-socket-files.md

```diff
@@ -6,66 +6,62 @@ alwaysopen: false
 categories: ["RS"]
 aliases: /rs/administering/installing-upgrading/configuring/change-location-socket-files/
 ---
-You can change the location of the socket files anytime, but to limit
-downtime, it is best to perform this change when you are
-[installing]({{< relref "/rs/installing-upgrading/downloading-installing.md" >}})
-or
-[upgrading]({{< relref "/rs/installing-upgrading/upgrading.md" >}})
-Redis Enterprise Software to version 4.5.0.-51 or higher. If this change
-needs to be performed immediately on an existing cluster, see the
-instructions below.
+There are two default locations for the socket files:
 
-## Changing the Location on an Existing Redis Enterprise Software (RS) Installation
+- `/tmp` - In clean installations of RS version lower than 5.2.2
+- `/var/opt/redislabs/run` - In clean installations of RS version 5.2.2 and higher
 
-You must be running RS version 4.5.0-51 or higher, and with the sock
-files location is still under the default location of /tmp. (This is
-where they go when install.sh is executed without -s flag.)
+When you upgrade from a RS version lower than 5.2.2 to 5.2.2 and higher, the socket files
+are not moved to the new location by default. During [installation]({{< relref "/rs/installing-upgrading/downloading-installing.md" >}})
+you can specify a custom location for the socket files, but after installation
```

Comment:
@RoeyPrat - I think we need to add the reason and the motivation for changing the socket path. Do you know what's the reason for it?

Reply:
--> Some customers have procedures which delete the /tmp content. So 
the motivation is some customers' operations which deletes all files under /tmp
@bmansheim

Reply:
@Paz-Yanover Added

Reply:
@bmansheim - good, thanks